# To demonstrate errors

zero_ = 0
a = 10

try:
    c = a/ zero_
except:
    print("An error {} encountered")
